apis/fluentbit/v1alpha2/plugins/output/open_telemetry_types.go

```diff
@@ -0,0 +1,86 @@
+package output
+
+import (
+	"fmt"
+	"github.com/fluent/fluent-operator/apis/fluentbit/v1alpha2/plugins"
+	"github.com/fluent/fluent-operator/apis/fluentbit/v1alpha2/plugins/params"
+)
+
+// +kubebuilder:object:generate:=true
+
+// OpenTelemetry is An output plugin to submit Metrics to an OpenTelemetry endpoint,
+// allows taking metrics from Fluent Bit and submit them to an OpenTelemetry HTTP endpoint.
+type OpenTelemetry struct {
+	// IP address or hostname of the target HTTP Server, default `127.0.0.1`
+	Host string `json:"host,omitempty"`
+	// TCP port of the target OpenSearch instance, default `80`
+	// +kubebuilder:validation:Minimum:=1
+	// +kubebuilder:validation:Maximum:=65535
+	Port *int32 `json:"port,omitempty"`
+	// Optional username credential for access
+	HTTPUser *plugins.Secret `json:"httpUser,omitempty"`
+	// Password for user defined in HTTP_User
+	HTTPPasswd *plugins.Secret `json:"httpPassword,omitempty"`
+	// Specify an HTTP Proxy. The expected format of this value is http://HOST:PORT. Note that HTTPS is not currently supported.
+	// It is recommended not to set this and to configure the HTTP proxy environment variables instead as they support both HTTP and HTTPS.
+	Proxy string `json:"proxy,omitempty"`
+	// Specify an optional HTTP URI for the target web server, e.g: /something
+	Uri string `json:"uri,omitempty"`
+	// Add a HTTP header key/value pair. Multiple headers can be set.
+	Header map[string]string `json:"header,omitempty"`
+	// Log the response payload within the Fluent Bit log.
+	LogResponsePayload *bool `json:"logResponsePayload,omitempty"`
```

Comment:
It seems that the parameter is not inserted here.

Reply:
I don't found the `parameter` field in this page(https://docs.fluentbit.io/manual/pipeline/outputs/opentelemetry), where I can find the ref?

Reply:
What I mean is that there should be code like this below
```
if o.LogResponsePayload != "" {
		kvs.Insert("logResponsePayload", o.LogResponsePayload)
	}
```

Reply:
Got, thanks~

Reply:
Done.